application/src/test/resources/sql/init.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test.members (
+                         id int,
```

Comment:
wenn wir die Datenbank auf servletDB setzen müsste hier auch `servletDB`.`members` 
diese schreibweise ist laut mysql doku: https://dev.mysql.com/doc/refman/8.0/en/create-table.html 

"The table name can be specified as db_name.tbl_name to create the table in a specific database. This works regardless of whether there is a default database, assuming that the database exists. If you use quoted identifiers, quote the database and table names separately. For example, write `mydb`.`mytbl`, not `mydb.mytbl`."